src/test/transaction_tests.cpp

```diff
@@ -174,9 +174,29 @@ unsigned int FillFlags(unsigned int flags)
 // Assumes that mapFlagNames contains all script verify flags.
```

Comment:
The doc is out of date now

Reply:
i can't tell what line you are commenting on

Reply:
Right, sorry about that.
```c
// Exclude each possible script verify flag from flags. Returns a set of these flag combinations
// that are valid and without duplicates. For example: if flags=1111 and the 4 possible flags are
// 0001, 0010, 0100, and 1000, this should return the set {0111, 1011, 1101, 1110}.
```